virt/managing_vms/virtual_disks/virt-migrating-storage-class.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+
+[id="virt-migrating-storage-class"]
+= Migrating VM disks to a different storage class
+
+include::_attributes/common-attributes.adoc[]
+:context: virt-migrating-storage-class
+
+toc::[]
```

Comment:
idk why this is not making a page TOC in either version of the preview, it looks formed correctly to me

Reply:
I don't know either, I checked again but this should be working.